packages/tupaia-web-server/src/routes/DashboardsRoute.ts

```diff
@@ -155,11 +213,28 @@ export class DashboardsRoute extends Route<DashboardsRequest> {
           .map(({ item }: { item: DashboardItem }) => ({
             ...item,
           })),
+        mailingLists: mailingLists
+          .filter(
+            (list: { dashboardId: string; entityCode: string; isSubscribed: boolean }) =>
+              list.dashboardId === dashboard.id,
+          )
+          .map(
+            ({
+              entityCode: mailingListEntityCode,
+              isSubscribed,
+            }: {
+              entityCode: string;
+              isSubscribed: boolean;
+            }) => ({
+              entityCode: mailingListEntityCode,
+              isSubscribed,
+            }),
+          ),
```

Comment:
```suggestion
          .filter((list) => list.dashboardId === dashboard.id)
          .map(
            ({
              entityCode: mailingListEntityCode,
              isSubscribed,
            }) => ({
              entityCode: mailingListEntityCode,
              isSubscribed,
            }),
          ),
```
I think TS can infer the types of functions in map/filter based on the array type